src/windows/list.cpp

```diff
@@ -579,12 +584,13 @@ std::vector<DeviceDescriptor> ListStorageDevices() {
     device.isSCSI = IsSCSIDevice(enumeratorName);
     device.isUSB = IsUSBDevice(enumeratorName);
     device.isCard = device.enumerator == "SD";
-    // device.isSystem = IsSystemDevice(hDeviceInfo, deviceInfoData);
-    device.isSystem = device.isSCSI && !device.isRemovable;
+    device.isSystem = !device.isRemovable &&
+      (device.enumerator == "SCSI" || device.enumerator == "IDE");
     device.isUAS = device.isSCSI && device.isRemovable &&
       !device.isVirtual && !device.isCard;
 
     if (GetDetailData(&device, hDeviceInfo, deviceInfoData)) {
+      device.isSystem = device.isSystem || IsSystemDevice(hDeviceInfo, &device);
```

Comment:
Hmmm, so (to paraphrase):
```
device.isSystem = (!device.isRemovable && (device.enumerator == "SCSI" || device.enumerator == "IDE")) || IsSystemDevice(hDeviceInfo, &device);
```
Are there any situations in which the Windows drive _won't_ be a non-removable SCSI or IDE drive? :wink: 

As (I think) I mentioned in #243 we need to remember that for Linux and Mac, we effectively still have:
```
device.isSystem = !device.isRemovable;
```

Reply:
> Are there any situations in which the Windows drive won't be a non-removable SCSI or IDE drive?

Yes – you can boot Windows off of pretty much anything, including those certified portable USB thumb drives.